lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.Model.rb

```diff
@@ -766,6 +766,9 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
       # Apply the HVAC efficiency standard -- !179D notes: autofan_turn_off apply to this one (both airLoop and ZoneHVAC)
       model_apply_hvac_efficiency_standard(model, climate_zone)
 
+      # NOTE: 179D Set the ACM schedule
+      model_apply_acm_hvac_availability_schedule(model)
```

Comment:
model_apply_hvac_efficiency_standard(model, climate_zone)

will already set the availability schedule, but we redo it for sure,a nd we also sert the UnitarySystem's Supply Air Fan Mode Operating Schedule.